src/main/java/core/basesyntax/ArrayList.java

```diff
@@ -1,48 +1,138 @@
 package core.basesyntax;
 
+import java.util.NoSuchElementException;
+
 public class ArrayList<T> implements List<T> {
+    private static final int DEFAULT_CAPACITY = 10;
+    private static final Object[] DEFAULT_CAPACITY_EMPTY_ELEMENTDATA = {};
+    private static final Object[] DEFAULT_CAPACITY_ELEMENTDATA = new Object[DEFAULT_CAPACITY];
```

Comment:
```suggestion
```
remove this, that is a static array
when you try creating 2 ArrayLists, and adding elements to them - you might see that values might be overridden
Because you would modify the same array in 2 different ArrayLists